platforms/quorum/configuration/reset-network.yaml

```diff
@@ -26,8 +26,18 @@
         kubernetes: "{{ item.k8s }}"
       loop: "{{ network['organizations'] }}"
 
-    # Delete Helmreleases and uninstall flux
-    # Change this role if you have new Helmreleases to delete 
+    # Uninstalling Flux is needed so that everything is clean
```

Comment:
Can you update the same in the corresponding role Readme.md?

Reply:
updated.